manifests/openshift-state-metrics-deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
             cpu: 20m
             memory: 40Mi
           requests:
-            cpu: 10m
```

Comment:
is there a reason your'e changing these requests/limits?

Reply:
This is because of [PR49](https://github.com/openshift/openshift-state-metrics/pull/49), but I'm guessing `make manifests` wasn't run then